flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/GenericInMemoryCatalogTest.java

```diff
@@ -731,32 +727,36 @@ public void testDropPartition() throws Exception {
 		catalog.createTable(path1, createPartitionedTable(), false);
 		catalog.createPartition(path1, createPartitionSpec(), createPartition(), false);
 
-		assertEquals(Arrays.asList(createPartitionSpec()), catalog.listPartitions(path1));
+		assertEquals(Arrays.asList(createPartitionSpec()), catalog.listAllPartitions(path1));
 
 		catalog.dropPartition(path1, createPartitionSpec(), false);
 
-		assertEquals(Arrays.asList(), catalog.listPartitions(path1));
+		assertEquals(Arrays.asList(), catalog.listAllPartitions(path1));
 	}
 
 	@Test
 	public void testDropPartition_TableNotExistException() throws Exception {
```

Comment:
We need to update the unit test names to reflect what case the test is actually about. E.g. this UT can be renamed to something like `testDropParition_tableNotExist()` or `testDropParition_ParitionNotExistException_tableNotExist()`.

Same thing for the following changed unit tests in this file.